### PR TITLE
x86_cpu_flags: update script "flag_disable"

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 2065031

Check the disable function with random flag

Signed-off-by: nanliu <nanliu@redhat.com>

### CODE REVIEW

**Comment by nanliu-r** [comment]
- (10/10) repeat10.Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.7.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.others.flag_disable.q35: PASS (52.61 s)
RESULTS    : PASS 9 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 1

**Comment by nanliu-r** [comment]
@PaulYuuu Could you please help to review this? Thanks a lot!

File: qemu/tests/x86_cpu_flag_disable.py [hunk 1]
```diff
@@ -17,8 +19,12 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment.
     """
-
+    flags_list = params["flags_list"].split()
```
Comment (PaulYuuu) [inline qemu/tests/x86_cpu_flag_disable.py:22]:
```suggestion
    flags_list = params.objects("flags_list")
```

File: qemu/tests/x86_cpu_flag_disable.py [hunk 2]
```diff
@@ -17,8 +19,12 @@ def run(test, params, env):
...
+    params["flags"] = random.choice(flags_list)
+    params["no_flags"] = params["flags"]
```
Comment (PaulYuuu) [inline qemu/tests/x86_cpu_flag_disable.py:24]:
```suggestion
    params["flags"] = params["no_flags"] = random.choice(flags_list)
```

**Comment by nanliu-r** [comment]
@PaulYuuu Thanks and update, please help to review again.

\ (10/10) repeat10.Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.7.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.others.flag_disable.q35: PASS (53.02 s)
RESULTS    : PASS 10 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

**Review by PaulYuuu — APPROVED** [review]

**Comment by nanliu-r** [comment]
> * x86_cpu_flags.others.flag_disable

@PaulYuuu  Thanks and update.

 (10/10) repeat10.Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.others.flag_disable.q35: PASS (32.91 s)
RESULTS    : PASS 9 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 1

**Pull request merged**